ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/event/RenameNodeEvent.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.event;
+
+import com.google.gwt.event.shared.GwtEvent;
+
+import org.eclipse.che.ide.project.node.ItemReferenceBasedNode;
+
+/**
+ * Event that notifies of renaming provided
+ *
+ * @author Igor Vinokur
+ */
+public class RenameNodeEvent extends GwtEvent<RenameNodeEventHandler> {
```

Comment:
see https://github.com/codenvy/che-core/pull/685#discussion_r51446069
rename event and handler in the past tense